core/src/main/java/org/kiwix/kiwixmobile/core/utils/Constants.kt

```diff
@@ -19,6 +19,7 @@ package org.kiwix.kiwixmobile.core.utils
 
 const val TAG_KIWIX = "kiwix"
 const val CONTACT_EMAIL_ADDRESS = "android@kiwix.org"
+const val KIWIX_SUPPORT_URL = "https://www.kiwix.org/support"
```

Comment:
This is used from precisely 1 place, you can keep the constant in CoreMainActivity